mmv1/products/cloudidentity/Group.yaml

```diff
@@ -128,6 +139,30 @@ properties:
     description: |
       An extended description to help users determine the purpose of a Group.
       Must not be longer than 4,096 characters.
+  - !ruby/object:Api::Type::String
+    name: 'dynamicGroupQuery'
```

Comment:
Wouldn't it be more in the style of other APIs to translate this to `dynamic_group_query`? I think that would fit the style of the culture better.